src/schema/objects/metadata.yaml

```
@@ -2825,6 +3023,55 @@ ScatterFraction:
     type: number
     minimum: 0
     maximum: 100
+ScreenDistance:
+  name: ScreenDistance
+  display_name: Screen Distance
+  description: |
+    Screen distance in cm
+    (for example `60` for a screen distance of 60 cm), if no screen use `n/a`.
```

Perhaps something more descriptive, like:
```suggestion
    Distance between the participant's eye and the screen. If no screen was used, use `n/a`.
```